server.py

```diff
@@ -19,15 +19,15 @@
 
 @app.route('/api/comments', methods=['GET', 'POST'])
 def comments_handler():
-    with open('comments.json', 'r') as f:
+    with open('comments.json', 'r',encoding='utf-8') as f:
```

Comment:
This and below are fine but please add a space after the comma (for pep8 style) and also please remove all of the code related to your implementation.